src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -177,6 +177,10 @@ def is_old_task(self, task):
         return False
 
     def is_mozilla_central_task(self, task):
+        if 'MH_BRANCH' not in task['task']['payload']['env']:
```

Comment:
Oops, did this happen on staging? Do you know under which circumstances `MH_BRANCH` is missing?

Reply:
> Oops, did this happen on staging? 

Yes :(

> Do you know under which circumstances MH_BRANCH is missing?

Unfortunately not, as Sentry is not working for shipit-pulse-listener for some reason (CC @garbas @La0).
With the logging I've added here we should know more once it happens again.